ow_lander/src/ow_lander/actions.py

```diff
@@ -425,6 +416,39 @@ def execute_action(self, goal):
     self._set_aborted("Timed out waiting for point cloud")
 
 
+class CameraSetExposureServer(ActionServerBase):
+
+  name          = 'CameraSetExposure'
+  action_type   = ow_lander.msg.CameraSetExposureAction
+  goal_type     = ow_lander.msg.CameraSetExposureGoal
+  feedback_type = ow_lander.msg.CameraSetExposureFeedback
+  result_type   = ow_lander.msg.CameraSetExposureResult
```

Comment:
I believe these (i.e. the CameraSetExposure action) should be defined in the `owl-msgs` package.  Though perhaps there is an argument for having them in `ow_lander` for now..

Reply:
In today's tagup we discussed this issue, and it's fine to leave these as is, so please ignore my last comment.